File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex12.java
//Importando Scanner
import java.util.Scanner;

public class Ex12 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	System.out.println("Salário a receber\n\n");
	
	//vars
	double salarioAtual, gratificacao, imposto, salarioFinal;
	
	//Entrada
	System.out.print("Insira o salário do funcionário: R$ ");
	salarioAtual = input.nextDouble();
	
	//Processamento
	imposto = salarioAtual * 0.07;
	
	if ( salarioAtual <= 350 )
		gratificacao = 100;
	else if ( salarioAtual <= 600 )
		gratificacao = 75;
	else if ( salarioAtual <= 900 )
		gratificacao = 50;
	else
		gratificacao = 35;
	
	salarioFinal = salarioAtual + gratificacao - imposto;
	
	//Saída
	System.out.printf("Salário a receber: R$ %.2f", salarioFinal);
		
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex12.c
#include <stdio.h>

main()
{
    printf("\n\n");

    //vars
    float salarioI, imposto, gratificacao, salarioF;

    //Entrada
    printf("Insira o salario do funcionario: ");
    scanf("%f",&salarioI);

    //Processamento
    imposto = salarioI * 0.07;

    if( salarioI <= 350 )
        gratificacao = 100;
    else if ( salarioI <= 600 )
        gratificacao = 75;
    else if ( salarioI <= 900 )
        gratificacao = 50;
    else
        gratificacao = 35;

    salarioF = salarioI - imposto + gratificacao;

    //Saída
    printf("Salario a receber: R$%.2f", salarioF);

}
